app/lib/features/chat_ng/widgets/events/chat_event_item.dart

```diff
@@ -30,36 +30,42 @@ class ChatEventItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final eventType = item.eventType();
+    final metadata = MessageMetadata(
```

Comment:
As said above, I think this place should decide _directly_ which children to render, rather than have that intelligence moved into the chat-bubble.